src/main/java/org/mybatis/dynamic/sql/select/render/PagingModelRenderer.java

```diff
@@ -25,10 +26,12 @@
 public class PagingModelRenderer {
     private RenderingStrategy renderingStrategy;
     private PagingModel pagingModel;
+    private AtomicInteger sequence;
 
     private PagingModelRenderer(Builder builder) {
         renderingStrategy = Objects.requireNonNull(builder.renderingStrategy);
         pagingModel = Objects.requireNonNull(builder.pagingModel);
+        sequence = builder.sequence().orElseGet(() -> new AtomicInteger(1));
```

Comment:
Please use the same pattern as the other parameters: sequence = Objects.requireNonNull(builder.sequence)